mitodl/open-discussions
# More intelligent OLX ingestion, and ingest static files for xPRO

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/BzzuItbZ/62-index-xpro-static-files-in-addition-to-olx

#### What's this PR do?
 - Uses the `display_name` on the vertical for the title of the `ContentFile`
 - Ingest static file content after passing through tika, using the file extension to indicate file type to tika

#### How should this be manually tested?
 - Run `./manage.py backpopulate_xpro_data` and `./manage.py backpopulate_xpro_files`.
 - In a Django shell query for `ContentFile.objects.filter(content_type='file', run__platform='xpro')`. You should see many objects which represent static file content.

Note that none of the `ContentFile`s will have a `vertical` `content_type` because it looks like none of the OLX data for courses which contain verticals are also in the courses API for xPRO
